tensorboard/webapp/dev_assets/BUILD

```diff
@@ -6,7 +6,7 @@
 # //tensorboard/webapp:index.html, we had to create the dev_assets version in
 # a different package. Once we can get rid of zip asset provider and a way to
 # alias asset, we would not need the enitrely different Bazel package.
-load("//tensorboard/defs:defs.bzl", "tf_dev_js_binary", "tf_js_binary", "tf_ng_module")
+load("//tensorboard/defs:defs.bzl", "tf_dev_js_binary", "tf_ng_module")
```

Comment:
Things here look extraneous and incorrect. Please fix the commit history and remove unnecessary changes.

Reply:
Should be done.